grunt-configs/sass.js

```diff
@@ -50,6 +50,17 @@ module.exports = function(grunt, options) {
                     return dest + src.replace('scss', 'css');
                 }
             }]
+        },
+        inline: {
+            files: [{
+                expand: true,
+                cwd: sassDir,
+                src: ['inline/*.scss'],
+                dest: options.staticTargetDir + 'stylesheets/',
+                rename: function(dest, src) {
+                    return dest + src.replace('scss', 'css');
+                }
+            }]
```

Comment:
Always hesitant to add more complexity to our pipeline, because it's already so complex. Could we somehow just make `sass:modern` also cover these?